.ado/templates/e2e-testing-uwp.yml

```diff
@@ -0,0 +1,42 @@
+steps:
+  - task: DownloadSecureFile@1
+    name: UwpCertificate
+    inputs:
```

Comment:
Interesting, you don't need to specify some path to search against?

Reply:
I added it to our Pipeline's "Secure Files" library in ADO. From there, I can just use the name I added it under to download on the agent machine.

Reply:
Nice!